src/arch/x86/object/interrupt.c

```diff
@@ -175,6 +175,9 @@ exception_t Arch_decodeIRQControlInvocation(word_t invLabel, word_t length, cte_
             return EXCEPTION_SYSCALL_ERROR;
         }
 
+#ifdef CONFIG_IOMMU
+        vtd_create_irte(handle, vector, pci_bus, pci_dev, pci_func);
```

Comment:
Does CONFIG_IOMMU imply VT-d support? What if the CPU has IOMMU support, but no VT-d support?

Reply:
We don't yet support AMD-V, so yes, CONFIG_IOMMU implies VT-d